### ISSUE TITLE
Arrow Vortex Link is not cooperative/dead

### ISSUE THREAD

**Issue by bloodcloak-beatsaber** [beat-saber-modding-group/wiki#25]
In the basic audio page Tool Assisted BPM Calculation section. 
https://github.com/beat-saber-modding-group/wiki/blob/master/wiki/mapping/basic-audio.md#tool-assisted-bpm-calculation

I am having difficulty accessing the arrow vortex website for the download. Have been getting a 552 error. 

How would we go about providing this tool? Hosting on the wiki or discord?

**Comment by bloodcloak-beatsaber** [comment]
I was finally able to access the download page.
My experience with the current link is after clicking and redirected to cookie check, I had to wait a few min before being dumped onto their forum.

I then reclicked the current wiki link and was brought to the info page with the download. 
I discussed with Helen and we decided to link the direct download instead.

Problem though, even the direct download from the website hits the cookie check in addition to the download being a rar file.
They did however post a mirror download. Unfortunately it is Mediafire, a site I think is a bit unsafe for the average user with the misleading download ads.

These are the options I have thought up
1. Only have the direct download linked and note that it might take a while to reach.
2. Also link the mirror with a disclaimer about the safety of the website
3. Rehost on a safe platform (the wiki itself/discord/etc) in zip form instead of rar.

Option 1 and 2 would include another note on the file being rar and needs additional software such as 7zip to unpack.

Comments @megalon @williums ?
